packages/reaction-core/server/main.js

```diff
@@ -23,6 +23,17 @@ _.extend(ReactionCore, {
     ReactionRegistry.createDefaultAdminUser();
     // hook after init finished
     ReactionCore.Hooks.Events.run("afterCoreInit", this);
+
+    // Make sure this event is registered last.
+    ReactionCore.Hooks.Events.add("orders/sendNotification", (result) => {
+      if (!result) {
+        //  If event was successfully captured the result will be falsy.
+        //  No-op
+      } else {
+         ReactionCore.Log.warn("Package email-notifications seems not being " +
```

Comment:
This should probably be a `fatal` and the correct English would "seems to not be installed. Try to fix it with....". Is the package not being installed the main reason this doesn't work? (being that it will be installed by default)